File: src/main/java/com/projectswg/forwarder/resources/networking/packets/SeriousErrorAcknowledge.java
package com.projectswg.forwarder.resources.networking.packets;

import java.nio.ByteBuffer;

public class SeriousErrorAcknowledge extends Packet {
	
	public SeriousErrorAcknowledge() {
		
	}
	
	public SeriousErrorAcknowledge(ByteBuffer data) {
		decode(data);
	}
	
	public void decode(ByteBuffer data) {
		data.position(2);
	}
	
	public ByteBuffer encode() {
		ByteBuffer data = ByteBuffer.allocate(2);
		addNetShort(data, 0x1D);
		return data;
	}
	
}

File: src/main/java/com/projectswg/forwarder/resources/networking/packets/SeriousErrorAcknowledge.kt
package com.projectswg.forwarder.resources.networking.packets

import com.projectswg.common.network.NetBuffer


class SeriousErrorAcknowledge : Packet {
	
	constructor()
	constructor(data: NetBuffer) {
		decode(data)
	}
	
	override fun decode(data: NetBuffer) {
		data.position(2)
	}
	
	override fun encode(): NetBuffer {
		val data = NetBuffer.allocate(2)
		data.addNetShort( 0x1D)
		return data
	}
	
}
